GCM/App/init/initializers/ForgotPasswordInitializer.java

```diff
@@ -1,21 +0,0 @@
-package init.initializers;
```

Comment:
If you see that you deleted 523490 files that you don't recognize than you probably did something wrong